components/automate-ui/src/app/pages/+compliance/shared/reporting/report-query.service.ts

```diff
@@ -44,15 +44,21 @@ export class ReportQueryService {
   }
 
   getReportQuery(): ReportQuery {
-    return this.state.getValue();
+    const reportQuery = this.state.getValue();
+    return {
+      startDate: reportQuery.startDate.clone(),
+      endDate: reportQuery.endDate.clone(),
+      interval: reportQuery.interval,
+      filters: Object.assign([], reportQuery.filters)
```

Comment:
Cleaner pattern for cloning that we (auth team) have been introducing in recent months--use the spread operator:
```suggestion
      filters: [...portQuery.filters]
```